mdaviz/select_fields_table_view.py

```diff
@@ -203,31 +162,30 @@ def to_datasets_mpl(detsDict, selections):
     )
 
     # y_axis is the list of row numbers
-    y_names = []
+    y_names_with_units = []
+    y_names_with_file_units = []
     for y_axis in selections.get("Y", []):
         y = detsDict[y_axis]
         y_data = y.data
+        # y labels:
         y_units = utils.byte2str(y.unit) if y.unit else "a.u."
-        y_name = utils.byte2str(y.name) + f" ({y_units})"
-        y_names.append(y_name)
-
+        y_name = utils.byte2str(y.name)
+        y_name_with_units = y_name + "  (" + y_units + ")"
+        y_name_with_file_units = fileName + ": " + y_name + "  (" + y_units + ")"
+        y_names_with_units.append(y_name_with_units)
+        y_names_with_file_units.append(y_name_with_file_units)
+        # append to dataset:
         ds, ds_options = [], {}
-        ds_options["label"] = f"{y_name}"
+        ds_options["label"] = y_name_with_file_units
         ds = [x_data, y_data] if x_data is not None else [y_data]
         datasets.append((ds, ds_options))
-
-    title = ""
-    if len(y_names) == 1:
-        title = f"{y_name} vs {x_name}"
-    elif len(y_names) <= 3:
-        title = "(" + " ,  ".join(y_names) + f")\n vs {x_name}"
-    else:
-        title = "(" + " ,  ".join(y_names[0:3]) + f", ...)\n vs {x_name}"
+    now = str(datetime.datetime.now()).rsplit(".")[0]
```

Comment:
There must be a natural way with datetime to not show fractional seconds.  [Perhaps maybe](https://www.8bitavenue.com/get-current-datetime-without-milliseconds-in-python/):

```py
datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
```

Reply:
Was there a commit?

Reply:
done